# metis-project
Data science project for Metis course

The central dogma of biology describes the basic framework for how genetic information flows from a DNA sequence to messenger RNA (mRNA) to a protein product inside cells. This process of genetic information flowing from DNA to mRNA to protein is called gene expression or transcription, and then from mRNA to protein is called translation.

In scientific research, transcriptomics analysis of gene expression is a commonly used experimental technique used to investigate changes in the properties of cells and tissues, for example, between normal and diseased states, or when a stressor or abnormal environment is applied. These changes are then identify which genes and proteins are potentially involved to provide insight on cellular biology and / or help identify possible targets for intervention.

This approach involves the working assumption that changes in gene expression are representative of the downstream changes in protein expression and function (e.g., an increase in gene expression results in increased activity for the product of that gene). However, for the changes in gene expression to be functionally significant, comparable changes downstream in protein expression are required (and ultimately protein function which is also affected by post-translational activity).

This project therefore plans to compare the transcriptomics data (which measures gene expression) from The Cancer Genome Atlas (TCGA https://cancergenome.nih.gov/) with the proteomics data (measuring protein levels) for the matching samples from The Cancer Proteomic Atlas (TCPA http://tcpaportal.org/tcpa/) to determine the correlation between gene and protein expression, and the influence of cancer type and/or pathway on this relationship.
